models/aave/aave_ethereum_schema.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+
+models:
+ - name: aave_v2_ethereum_interest_rates
+   meta:
+     blockchain: ethereum
+     sector: interest

```

Comment:
i wonder if sector shouldn't be a more broad subject area like lending/borrowing? not sure exactly what we'd look to call it. we could also leave sector out completely and leave just project 'aave'